### DISCUSSION TITLE
Compiling

### DISCUSSION THREAD

**Discussion by ghost** [zmxhawrhbg/gdrpc#9] [Q&A]
Hello there!
How do you import files as a CMake on Visual Studio? I am pretty confused with this.

**Comment by qimiko** [comment]
Please refer to [the Visual Studio documentation](https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-160)

**Comment by ghost** [comment]
Alright, after doing that. And I need to download the git modules. "The term 'git' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of
the name, or if a path was included, verify that the path is correct and try again." (Did this in the VS 2019 Terminal.)

**Reply by qimiko** [reply]
You must download [Git for Windows](https://git-scm.com/download/win). Make sure it gets put into your path

**Comment by ghost** [comment]
Ok so it says something not found? I also tried using git clone.. Can you
make a Tutorial on this?

**Reply by qimiko** [reply]
It sounds like you haven't properly installed git or properly added it to your path. Unfortunately, I am no longer in a position where I can provide tutorials on the build process as both my release and development build process are very different from what would be standard.

**Comment by ghost** [comment]
alright, just to clarify. should i use git clone and the repo?

**Comment by ghost** [comment]
ok so got it to only compile  discord-rpc.dll and gdrp.dll but what bout zliib or smth?

**Reply by qimiko** [reply]
zlib is manually made, I would just grab the copy from the releases tab as whatever you would make would be the same

**Comment by ghost** [comment]
alright was about to compile now and saw you realeased gdps support. thanks
